src/modules/CC1101/CC1101.h

```diff
@@ -745,6 +758,14 @@ class CC1101: public PhysicalLayer {
     */
     int16_t setRxBandwidth(float rxBw);
 
+     /*!
+      \brief calculates and sets Rx bandwidth based on the freq, baud and freq uncertainty.
+      \Reimplement of atlas0fd00m (RfCat) CalculatePktChanBw function. 
+      \Modified for worse ppm with the CC1101, and adjusted for the supportted CC1101 bw.
+      \returns \ref status_codes
+    */
+    int16_t CalcRxBandwidth();
+
```

Comment:
The name of this method seems misleading - I would expect a method that is called "calc" to calculate something, not calculate and set it. I propose to change this to `autoSetRxBandwidth`.

Also, the leading `\` is not supposed to be on every line - it denotes commands for Doxygen, which generates documentation from these comments. So the leading `\` should be removed from the second and third line of the comment.

Finally, the capitalization is incorrect, RadioLib uses `camelCase` in public methods.

Reply:
Also, public methods must be added to keywords.txt - be careful that the file uses true tabs and not spaces!

Reply:
All 3 complete. 

I am editing on github itself, so hopefully when setting the Tab drop down it is an actual tab.